porting/test.py

```diff
@@ -95,30 +95,30 @@ def run_and_compare(inputfile, args=""):
 
 def testcase(inputfile, args_to_try=DEFAULT_ARGS):
     """Run both scripts on inputfile with various arguments."""
-    print inputfile,
+    print(inputfile),
     try:
         for args in args_to_try:
-            print ".",
+            print(".", end='')
```

Comment:
I think this should be `end=' '` to preserve behavior.

(Not that it matters much.)